dsl/pace/dsl/dace/dace_config.py

```diff
@@ -7,10 +7,10 @@
 from pace.util.communicator import CubedSphereCommunicator
 
 
-# TODO (floriand): Temporary deactivate the distributed compiled
-# until we deal with the Grid data inlining during orchestration
-# See github issue #301
-DEACTIVATE_DISTRIBUTED_DACE_COMPILE = True
+# This can be turned on to revert compilation for orchestration
+# in a rank-compile-itself more, instead of the distributed top-tile
+# mechanism.
+DEACTIVATE_DISTRIBUTED_DACE_COMPILE = False
```

Comment:
If this value changes in practice, please make sure this is called out in the first message and in the body of the squashed merge commit. Maybe it's already there, I can't tell.

Reply:
Well this is really going back to the default - but keeping the flag for quick debug. I have it spelled out in the commit, not sure how far down